pkgs/test/make-binary-wrapper/default.nix

```diff
@@ -15,16 +15,25 @@ let
   makeGoldenTest = testname: runCommand "make-binary-wrapper-test-${testname}" env ''
     mkdir -p tmp/foo # for the chdir test
 
-    params=$(<"${./.}/${testname}.cmdline")
+    source=${lib.fileset.toSource {
+      root = ./.;
+      fileset = lib.fileset.unions [
+        (./. + "/${testname}.cmdline")
+        (./. + "/${testname}.c")
+        (lib.fileset.maybeMissing (./. + "/${testname}.env"))
+      ];
+    }}
```

Comment:
This would be clearer if extracted to another binding.

Reply:
I think this is fine, I don't think much is gained from having an intermediate Nix binding (we already assign it to a bash variable here)